README.md

```diff
@@ -255,7 +256,7 @@ An example logger exporter is below:
 
 type exporter struct{}
 
-func (e *exporter) Export(vd *stats.ViewData) {
+func (e *exporter) ExportView(vd *stats.ViewData) {
```

Comment:
Did we forget to perhaps do this in the internal/readme/source.md file too? I don't know how the embedmd stuff works :)

Reply:
I assumed someone changed README rather than internal/readme/source.md.